src/frontend/src/content/docs/ja/whats-new/aspire-13-1.mdx

```diff
@@ -0,0 +1,591 @@
+</Aside>
+
+## 🤖 AI コーディングエージェント向け MCP
+
+Aspire 13.1 では、MCP（Model Context Protocol）との統合により、AI コーディングエージェントを包括的にサポートします。お好みの AI コーディングツールを設定することで、Aspire アプリケーションを理解し、操作できるようになります。
+
+### プロジェクト用に MCP を初期化する
+
+新しい `aspire mcp init` コマンドは、開発環境を検出し、対応する AI コーディングエージェント向けに MCP サーバーを設定します:
+
+```bash title="Bash — MCP 構成を初期化"
+aspire mcp init
+```
+
+<LearnMore>
+コマンド リファレンス: [`aspire mcp init`](/ja/reference/cli/commands/aspire-mcp-init/).
+</LearnMore>
+
+このコマンドでは、簡潔な 2 ステップの選択プロセスが表示されます:
+
+```bash
+# どのエージェント環境を設定しますか？
+Which agent environments do you want to configure?
+[ ] Configure VS Code to use the Aspire MCP server
+[ ] Configure GitHub Copilot CLI to use Aspire MCP server
+[ ] Configure Claude Code to use Aspire MCP server
+[ ] Configure Open Code to use Aspire MCP server
+
+# 追加で有効にするオプションはどれですか？
+Which additional options do you want to enable?
+[ ] Create an agent instructions file (AGENTS.md)
+[ ] Configure Playwright MCP server
+```
+
+### AI エージェント向け MCP ツール
+
+MCP 経由で接続された AI コーディングエージェントは、Aspire の統合機能を直接検出して利用できるようになります:
+
+- **`list_integrations`** — 利用可能な Aspire Hosting の統合（Redis、PostgreSQL、Azure サービスなど）を返します。
+- **`get_integration_docs`** — 特定の統合パッケージに関する詳細なドキュメントを取得します。
+- **`list_apphosts`** — ワークスペース内で検出されたすべての AppHost プロジェクトを一覧表示します。
+- **`select_apphost`** — 複数の AppHost が存在する場合に、対象の AppHost にコンテキストを切り替えます。
+
+### ダッシュボードの MCP 統合
+
+AppHost に接続されている場合、ダッシュボードは MCP エンドポイントを公開し、AI エージェントが実行中のアプリケーションから直接、リソースの状態確認、ログの参照、トレースの調査を行えるようになります。
+
+<LearnMore>
+詳細については、 [MCP サーバー統合](/ja/dashboard/mcp-server/) をご参照ください。
+</LearnMore>
+
+## 🛠️ CLI の機能強化
+
+### チャネル選択と永続化
+
+`aspire new` および `aspire init` コマンドで `--channel` オプションが利用可能になり、使用する Aspire のバージョンチャネルを明示的に選択できるようになりました:
+
+```bash title="Bash — チャネルの選択"
+aspire new aspire-starter --channel preview
+aspire init --channel stable
+```
+
+<LearnMore>
+コマンド リファレンス: [`aspire init`](/ja/reference/cli/commands/aspire-init/#options).
+</LearnMore>
+
+`aspire update --self` で CLI を更新すると、選択したチャネルは `~/.aspire/globalsettings.json` に保存され、以後のプロジェクト作成時の既定値として使用されます。
+
+### 自動インスタンス検出
+
+すでにインスタンスが起動している状態で `aspire run` を実行した場合でも、Aspire 13.1 は実行中のインスタンスを自動的に検出し、終了させます。
+
+### インストールパス オプション
+
+インストール スクリプトで `--skip-path` がサポートされ、PATH を変更せずに CLI をインストールできるようになりました:
+
+<OsAwareTabs syncKey="terminal">
+  <Code
+    slot='unix'
+    lang="bash"
+    code="curl -fsSL https://aspire.dev/install.sh | bash -s -- --skip-path"
+  />
+  <Code
+    slot='windows'
+    lang="powershell"
+    code='iex "& { $(irm https://aspire.dev/install.ps1) } -SkipPath"'
+  />
+</OsAwareTabs>
+
+<LearnMore>
+詳細については [Aspire CLI インストールオプション](/ja/reference/cli/install-script/#options) をご確認ください。
+</LearnMore>
+
+## 📊 ダッシュボードの改善
+
+### 専用の Parameters タブ
+
+リソースページに専用の **Parameters** タブが追加され、リソースの詳細画面から離れることなく、設定パラメーターをより簡単に確認・管理できるようになりました。
+
+<Image src={resourcesParametersView} alt="Resources parameters view" title="Resources parameters view" />
```

Comment:
Alt text could also be translated. It's shown on screen as well when you click to zoom on an image.